src/components/attendee/MiniProfile/MiniProfile.tsx

```diff
@@ -0,0 +1,20 @@
+import { ElementId } from "types/id";
+
+import { useProfileModalControls } from "hooks/useProfileModalControls";
+
+import { MiniProfileModal } from "./components";
+
+interface MiniProfileProps {
+  parentComponent: ElementId;
+}
+export const MiniProfile: React.FC<MiniProfileProps> = ({
+  parentComponent,
+}) => {
+  const { selectedUserId, selectedElementId } = useProfileModalControls();
+
+  if (!(selectedElementId && selectedElementId === parentComponent)) {
+    return <></>;
+  }
+
+  return <MiniProfileModal userId={selectedUserId} />;
```

Comment:
I will list few options, in case any one or combination of few makes the code here more readable

option 1: ` ! ( A && B === C )` can be converted to ` A || B !== C`

option 2: The `if-else` can be converted to `? :`

option 3: 
```ts
if ( ! condition ){
  // something
} else {
  // something else
}
```

can be converted into

```tsx
if ( condition ){
  // something else
} else {
  // something something
}
```

Reply:
used the 1st option: https://github.com/sparkletown/sparkle/pull/2953/commits/a52527d7fce74b5f15447723260cc7715c8614b3